sky/server/server.py

```diff
@@ -20,6 +20,7 @@
 import fastapi
 from fastapi.middleware import cors
 import starlette.middleware.base
+from starlette.middleware.base import BaseHTTPMiddleware
```

Comment:
import module instead of class. See google style guide for python

Reply:
Updated